src/wallet/wallet.ts

```diff
@@ -634,6 +634,7 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
    */
   async pollForWalletStatus(): Promise<void> {
     return new Promise((resolve, reject) => {
+      // eslint-disable-next-line consistent-return -- Polling function does not need to return always
```

Comment:
Instead of disabling the rule, why not change the return statements?

`return resolve();` to
```ts
resolve();
return;
```

Then we can add a return on the end of the method, creating a consistent return in this method.